src/components/pages/TicketsDashboard/TicketsDashboard.js

```diff
@@ -0,0 +1,127 @@
+import React, { useEffect, useState } from 'react';
+import axiosWithAuth from '../../../utils/axiosWithAuth';
+import { Statistic, Row, Col, Table } from 'antd';
+// TODO: update page styling and functionality, see wireframes/NewDesignProposition/AdminWireframes/Dashboard
+
+const columns = [
+  {
+    title: 'Ticket ID',
+    dataIndex: 'ticketID',
+    filters: [
+      {
+        text: 'Example',
+        value: 'Example',
+      },
+      {
+        text: 'Jim',
+        value: 'Jim',
+      },
+      {
+        text: 'Submenu',
+        value: 'Submenu',
+        children: [
+          {
+            text: 'Green',
+            value: 'Green',
+          },
+          {
+            text: 'Black',
+            value: 'Black',
+          },
+        ],
+      },
+    ],
+    // specify the condition of filtering result
+    // here is that finding the name started with `value`
+    onFilter: (value, record) => record.ticketID.indexOf(value) === 0,
+    sorter: (a, b) => a.ticketID - b.ticketID,
+    sortDirections: ['descend'],
+  },
+  {
+    title: 'Message',
+    dataIndex: 'message',
+    filters: [
+      {
+        text: 'London',
+        value: 'London',
+      },
+      {
+        text: 'New York',
+        value: 'New York',
+      },
+    ],
+
+    onFilter: (value, record) => record.ticketType.indexOf(value) === 0,
+  },
+  {
+    title: 'Date Submitted',
+    dataIndex: 'dateSubmitted',
+    defaultSortOrder: 'descend',
+    sorter: (a, b) => a.dateSubmitted < b.dateSubmitted,
+  },
+];
+
+function onChange(pagination, filters, sorter, extra) {
+  // console.log('params', pagination, filters, sorter, extra);
```

Comment:
do we need this func? if all it's doing is `console.logging`?

Reply:
This component was dashboard before I changed it to ticketsdashboard, apologies but I didnt pay attention to the code as It was done by somebody else, I just changed Its directory and name, I removed all the console logs and functions that are not working there